apps/system/test/unit/lockscreen/widgets/clock/lockscreen_clock_widget_test.js

```diff
@@ -19,21 +19,20 @@ suite('LockScreenClockWidget > ', function() {
   });
 
   test(`it would update the clock`, function() {
-    window.Date = this.sinon.stub().returns('dummy-date');
-    navigator.mozL10n = {
-      DateTimeFormat: function() {
-        return {
-          'localeFormat': navigator.mozL10n.dummyLocaleFormat
-        };
-      },
-      dummyLocaleFormat: this.sinon.stub().returns('dummy-locale-format'),
-      get: this.sinon.stub().returns('dummy-l10n-string')
-    };
     var mockThis = {
       logger: {
         debug: function() {}
       },
-      _timeFormat: '%p',
+      timeFormatter: new Intl.DateTimeFormat('en-US', {
```

Comment:
So it's an API, right? Since you don't mock it. But I still wonder if JSHint let this pass because you don't declare the `global`, unless our JSHint config had been modified in another bug.